appveyor.yml

```diff
@@ -17,7 +17,12 @@ configuration:
 
 build:
   project: src/Everything.sln
-  
+
+before_test:
+  - ps: cl.exe /c /analyze /analyze:quiet- /W4 /analyze:autolog:ext sarif src\Sarif.FunctionalTests\ConverterTestData\PREfast\src\AdvancedAliasing\AA_IndexAlias.cpp; ls
```

Comment:
This is not a PowerShell script, so according to the [appveyor.yml Reference](https://www.appveyor.com/docs/appveyor-yml), it should either start with "`cmd: `", or omit the prefix (because that's the default).